tests/cdi/cdi_suite_test.go

```diff
@@ -0,0 +1,36 @@
+package cdi_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	ktests "kubevirt.io/kubevirt/tests"
+	_ "kubevirt.io/kubevirt/tests"
+)
+
+func TestCDI(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "CDI Suite")
+}
+
+var _ = BeforeSuite(func() {
+	out, err := ktests.RunOcCommand("get", "projects")
+	Expect(err).ToNot(HaveOccurred())
+	Expect(out).Should(ContainSubstring("golden-images"))
+
+	_, err = ktests.RunOcCommand("project", "golden-images")
+	Expect(err).ToNot(HaveOccurred())
+})
+
+var _ = AfterSuite(func() {
+	_, err := ktests.RunOcCommand("delete", "pvc", pvcName)
+	Expect(err).ToNot(HaveOccurred())
+
+	_, err = ktests.RunOcCommand("delete", "pod", "-l", "app=containerized-data-importer")
```

Comment:
Again, it is better to wait for the target pod to actually be deleted. All this is doing is testing that the api's response to DELETE is ok. There are many reasons why a pod is not deleted and so the wait needs a max time.